site/docs/api/create-container.md

```diff
@@ -0,0 +1,43 @@
+---
+title: createContainer
+parent: api
+---
+
+# createContainer
+
+Creates a single scoped container for use with [CSS Container Queries]. This avoids potential naming collisions with other containers.
```

Comment:
Nit: Should it say "single scoped container _name_"? Would it be fair to say this function by itself doesn't create a container?

Reply:
This is also in the changeset if you think it's worth changing.